src/query/interpret/eval.hpp

```diff
@@ -270,30 +270,28 @@ class ExpressionEvaluator : public ExpressionVisitor<TypedValue> {
   }
 
   TypedValue Visit(InListOperator &in_list) override {
-    TypedValue *_list_ptr = nullptr;
     TypedValue _list;
```

Comment:
No reason to have `_list`. This was used as a local temporary object that held a copy in case pointer was null.